src/Annotation/Generate.php

```diff
@@ -151,86 +151,78 @@ class Generate
     public $encryption_alias;
 
     /**
-     * @return string
+     * @return string|bool|null
```

Comment:
If you are planning to release this as a BC break, it really is worth considering whether we can take this with along.

Afaik a replace of `@AG\Generate(true)` to `@AG\Generate("public")` and `@AG\Generate(false)` to `@AG\Generate("none")` should be enough.

If it takes you too much time, please drop this remark.

Reply:
Will be fixed in the next commit I push. :+1: